packages/talker_dio_logger/lib/dio_logs.dart

```diff
@@ -7,6 +7,8 @@ import 'package:talker_dio_logger/talker_dio_logger.dart';
 const _encoder = JsonEncoder.withIndent('  ');
 const _hiddenValue = '*****';
 
+const kDioLogsTimeStampKey = '_talker_dio_logger_ts_';
```

Comment:
Please make ```kDioLogsTimeStampKey``` static const field of ```TalkerDioLogger```
Exporting global constants is not good practice for dart packages